manifests/blacklist.pp

```diff
@@ -1,10 +1,13 @@
 # ex: si ts=4 sw=4 et
 
 define shorewall::blacklist (
-    $type         = 'ipv4',
+    $action        = 'DROP',
+    $type          = 'ipv4',
+    $source        = 'all',
     $address       = '',
+    $dest          = '$FW',
     $proto         = '',
-    $port          = '',
+    $port          = [],
```

Comment:
What happens on line 15 when `$port` is an array? The docs for `validate_re` require the first argument to be a string.